Documentation/network-policies.md

```diff
@@ -0,0 +1,30 @@
+
+# Network policies
+
+[Network policies](https://kubernetes.io/docs/user-guide/networkpolicies/) allow you to restrict the ingress traffic between pods. 
+To keep your cluster safer, it's strongly recomended to enable network policies into prometheus namespace.
+
+# Example
+
+In this example we are closing all the inbound communication on the namespace monitoring and just allowing the necessary traffic.
+
+
+Follow the steps [here](http://docs.projectcalico.org/v1.5/getting-started/kubernetes/installation/) to install calico, also dont' forget to [enable network policy](https:/kubernetes.io/docs/user-guide/networkpolicies) in you k8s cluster.
+
+Once you've done that, you should annotate your namespace to deny all the ingress (inboud) traffic.
+
+``` 
+$ kubectl annotate ns monitoring "net.beta.kubernetes.io/network-policy={\"ingress\":{\"isolation\":\"DefaultDeny\"}}"
+namespace "monitoring" annotated
+```
+
+In this step you can't reach any port in your pod, so let's apply this network policies examples to allow the necessary traffic.
+
+```
```

Comment:
As this will be rendered onto coreos.com at some point, we'll need to render the markdown files into this document. It would be enough if you just do a header for each of them like this:

```markdown
#### Alertmanager
[embedmd]:# (../example/networkpolicies/alertmanager.yaml)
...
```

Note what I used above:
```markdown
[embedmd]:# (../example/networkpolicies/alertmanager.yaml)
```

That is used to embed the yaml files into the document. Once you add those lines and run `make docs` they will automatically be up to date with what is in the files. (we make sure the documentation is up to date by checking with the `make docs` in our CI pipeline)